src/equations/compressible_euler_multicomponent_2d.jl

```diff
@@ -1,67 +1,96 @@
 
 @doc raw"""
-    CompressibleEulerMulticomponentEquations2D
+    CompressibleEulerMulticomponentEquations2D(gammas, gas_constants)
```

Comment:
```suggestion
    CompressibleEulerMulticomponentEquations2D(; gammas, gas_constants)
```
To indicate that people should use the ctor with keyword arguments

Reply:
done in 4db20a5